version.json

```diff
@@ -1,6 +1,6 @@
 {
     "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
-    "version": "1.1",
+    "version": "2.0",
```

Comment:
Is this the update for the major version due the breaking change (comment above)?
If so, perfect.

Reply:
Yes